tests/test_lib_importotp.py

```diff
@@ -637,6 +638,6 @@ def test_00_gpg_decrypt(self):
         self.assertTrue("2F25BAF8645350BB" in pubkeys)
 
         r = GPG.decrypt(str(HALLO_PAYLOAD))
```

Comment:
I think we can safely remove the `str()` here, the payload is already a string.

Reply:
WIll do.

Reply:
done in d367ea32